stylesheets/styles.css

```diff
@@ -0,0 +1,290 @@
+/* * {
+  border: black 1px solid;
+} */
+
+body {
+  max-width: 75rem;
+  max-height: 100rem;
+}
+
+.container {
+  display: grid;
+  grid-template: 5% 10% 20% 15% 15% 20% 10% 5% /100%;
+  text-align: center;
+}
+
+.navigation-bar {
+  display: grid;
+  grid-template: 1fr / 1fr 3fr 1fr 1fr;
+}
+
+.navigation-bar li:nth-child(1) {
+  grid-column: 1/2;
+}
+
+.navigation-bar li:nth-child(2) {
+  grid-column: 3/4;
+}
+
+.navigation-bar li:nth-child(3) {
+  grid-column: 4/5;
+}
+
+li {
+  list-style-type: none;
+  font-size: 1rem;
+  color: indigo;
+  font-weight: bold;
+}
+
+nav li:hover {
+  font-size: 1.2rem;
+  color: teal;
+}
+
+h1, h2 {
+  font-family: 'Allerta Stencil', sans-serif;
+  color: indigo;
+  font-size: 4rem;
+}
+
+h2 {
+  font-size: 2rem;
+}
+
+.headline {
+  display: grid;
+  grid-template: 3fr 1fr 1.5fr / 100%;
+  max-height: 300px;
+}
+
+.headline h1 {
+  grid-row: 1 2;
+  font-family: 'Allerta Stencil', sans-serif;
+  color: indigo;
+}
+
+.headline sub {
+  grid-row: 2 3;
+  font-size: 1.75rem;
+}
+
+.headline a {
+  grid-row: 3 4;
+}
+
+.quote {
+  display: grid;
+  grid-template: 20% 40% 20% / 0.5fr 0.5fr 1fr 1fr 1fr 1fr 1fr;
+  font-style: italic;
+  color: indigo;
+}
+
+.quote blockquote {
+  grid-row: 2/3;
+  grid-column: 3/7;
+  font-size: 1rem;
+  line-height: 2rem;
+  text-align: left;
+}
+
+.quote p {
+  grid-row: 3/4;
+  grid-column: 6/8;
+}
+
+.why {
+  display: grid;
+  grid-template: 35% 65% / 1fr;
+}
+
+.why h2 {
```

Comment:
Used pseudo elements and class to format layout of the unordered list items.  Every element's format is defined from the header to the list item and layout is applied consistently.